README.md

```diff
@@ -62,11 +50,6 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
 
```

Comment:
- [ ] This section has no License. Kindly delete this section or add a License file with link. I strongly recommend to add .MIT License to the project root and add a link to it here 😉